cloudman/values.yaml

```diff
@@ -178,9 +178,17 @@ cloudlaunch:
       oidc_public_uri: "{{.Values.ingress.protocol }}://{{ .Values.global.domain | default (index .Values.ingress.hosts 0) }}/cloudman"
     postgresql:
       enabled: true
-      postgresDatabase: cloudman
-      postgresUser: cloudman
-      postgresPassword: some_pass
+      postgresqlDatabase: cloudman
+      postgresqlUsername: cloudman
+      postgresqlPassword: some_pass
```

Comment:
Shouldn't this be commented out? The auto-generation works now right?

Reply:
This would need to be changed in cloudlaunch-helm, I don't think it already works

Reply:
Done in https://github.com/CloudVE/cloudlaunch-helm/pull/7